src/util.ts

```diff
@@ -12,7 +12,7 @@ import isInteger from 'lodash/isInteger';
 import { decode as rlpDecode, encode as rlpEncode } from 'rlp';
 import { ecdsaRecover } from 'secp256k1';
 import { Calldata } from '.';
-import superagent from 'superagent'
+import cloneDeep from 'lodash/cloneDeep'
```

Comment:
Defined but not used

Reply:
[6575589](https://github.com/GridPlus/gridplus-sdk/pull/469/commits/6575589b71f6f6b8f63c273c13d3a8d9e523681c)